src/lib.rs

```diff
@@ -136,20 +138,36 @@ pub struct Coordinate3d {
     edges: Vec<Coordinate3d>,
 }
 
-//Translate a coordinate by a vector quantity
 impl Coordinate3d {
+    ///Translates a coordinate by a vector quantity
     fn translate(&mut self, vector: ColumnVector<3>) {
         self.pos += vector;
     }
+
+    ///Calculates the distance between two Coordinate3d's
+    fn distance(self, cord: Coordinate3d) -> f64 {
+        //pythagorean theorem
+        //sqrt((x1 + x2)^2 + (y1 + y2)^2 + (z1 + z2)^2)
+        let mut ans: f64 = 0.0;
+        for axis in 0..3 {
+            ans += (self.pos[(axis, 1)] + cord.pos[(axis, 1)]).powi(2);
+        }
+
+        ans = ans.sqrt();
+        ans
```

Comment:
Can't you just `ans.sqrt()` here?

Reply:
Fixed just a second ago in today's commit to PR_1